### PR TITLE
feat(HLS): Add the update period for HLS manifest

### PR DESCRIPTION
**Pull request by Iragne**
Create the update period for hls parse
still need to check it

Close https://github.com/shaka-project/shaka-player/issues/7505

### CODE REVIEW

File: externs/shaka/player.js [hunk 1]
```diff
@@ -1294,6 +1295,13 @@ shaka.extern.DashManifestConfiguration;
  *   https://www.akamai.com/blog/performance/-using-ll-hls-with-byte-range-addressing-to-achieve-interoperabi
  *   <br>
  *   Defaults to <code>true</code>.
+ * @property {number} updatePeriod
+ *   Override the updatePeriod of the manifest. The value is in second
```
Comment (joeyparrish) [inline externs/shaka/player.js:1299]: My only complaint about this PR is the way this is phrased.  I'm going to make an edit to clean up the grammar.

Comment (Iragne) [reply externs/shaka/player.js:1299]: Sorry my French

File: externs/shaka/player.js [hunk 2]
```diff
@@ -1179,10 +1179,10 @@ shaka.extern.xml.Node;
  *   <br>
  *   Defaults to <code>false</code>.
  * @property {number} updatePeriod
- *   Override the minimumUpdatePeriod of the manifest. The value is in second
- *   if the value is greater than the minimumUpdatePeriod, it will update the
- *   manifest less frequently. if you update the value during for a dynamic
- *   manifest, it will directly trigger a new download of the manifest
+ *   Override the minimumUpdatePeriod of the manifest. The value is in seconds.
```
Comment (joeyparrish) [inline externs/shaka/player.js:1182]: I realized much of this was copied from the DASH version, so I updated both.

File: externs/shaka/player.js (hunk 1 above)
Comment (joeyparrish) [reply externs/shaka/player.js]: No worries!  English is stupid and hard sometimes.  For example, why is it "sometimes" and not "some times"?  I'm not sure.  That's just native speaker intuition for me.  And yet, other times, my native speaker intuition leads me to make stupid or lazy mistakes of my own.

**Review by joeyparrish — APPROVED** [review]

**Comment by joeyparrish** [comment]
@Iragne, thank you for your contribution!

File: externs/shaka/player.js (hunk 2 above)
Comment (Iragne) [reply externs/shaka/player.js:1182]: Thanks a lot

**Pull request merged**